content/docs/2.12/concepts/scaling-deployments.md

```diff
@@ -66,6 +66,17 @@ spec:
           - type: Percent
             value: 100
             periodSeconds: 15
+    complexScalingLogic:                                    # Optional. Section to specify complex scaling logic
+      formula: {formula-for-fetched-metrics}                # Optional. Formula for calculation
+      target: {target-value-to-scale-on}                    # Optional. If metrics are anyhow composed together
+      externalCalculators:                                  # Optional. Section for user defined grpc calculation servers
+        - name: {name-of-ec}                                # Mandatory if externalCalculators section is given
+          url: {url-of-ec-server}                           # Mandatory if externalCalculators section is given
+          timeout: {time-to-connect}                        # Mandatory if externalCalculators section is given
+          certDirectory: ""                                 # Optional. Defaults to "", no certificates used
+        fallback:                                           # Optional. Section to specify ec-fallback options
+          replicas: 3                                       # Mandatory if fallback section is included
+          failureThreshold: 3                               # Mandatory if fallback section is included
```

Comment:
If this fallback is for the external server, I think that it's wrong indented

Reply:
the fallback is for external servers, if one of them fails, the fallback is applied, works the same like normal triggers. I was thinking if triggers have error, normal fallback is applied and this logic can still work, therefore if any of the server fail, that would be the fallback for it. But i agree, this could be simplified. I could simply call the same fallback function as the standard one if an error occurs in external servers

Reply:
ive got rid of the grpc server fallback entirely and everything is handled with the default fallback as it exists currently